training_record/models.py

```diff
@@ -188,3 +192,21 @@ def __str__(self):
         return str(_f('{}状态于{}由{}变为{}', self.record_id, self.time,
                       self.get_pre_status_display(),
                       self.get_post_status_display()))
+
+
+class CampusEventFeedback(models.Model):
+    '''CampusEventFeedback stores text-like feedback for records.'''
+    class Meta:
+        verbose_name = _('培训活动反馈')
+        verbose_name_plural = _('培训活动反馈')
+
+    create_time = models.DateTimeField(verbose_name=_('创建时间'),
+                                       auto_now_add=True)
+    record = models.ForeignKey(Record, verbose_name=_('培训记录'),
```

Comment:
I think this should be an `OneToOne` field instead of `ForeignKey`, since a campus event record can have one feedback at most.